x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/index.tsx

```diff
@@ -126,6 +127,7 @@ const HostsTableComponent = React.memo<HostsTableProps>(
             updateHostsSort({
               sort,
               hostsType: type,
+              tableType,
```

Comment:
Is this leftover? I see you backed out the changes in the [action and reducer](https://github.com/elastic/kibana/commit/1d7b42e225d99ef5a3acc3520f3031f6fd1bdc0e).